concepts/strings/.meta/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
+  "blurb": "Learn about string formatting.",
```

Comment:
See above comment.

Reply:
I've approved with an update to this blurb in mind :)

Reply:
I tried to improve it